transactions/transactions.py

```diff
@@ -99,6 +99,35 @@ async def sign(self, ctx, user: discord.Member, team_name: str):
             except Exception as e:
                 await ctx.send(e)
 
+
+    @commands.guild_only()
+    @commands.command()
+    @checks.admin_or_permissions(manage_roles=True)
+    async def resign(self, ctx, user: discord.Member, team_name: str):
+        """Re-signs a user to a given team, and if necessary, reapplys roles before posting to the assigned channel"""
+        franchise_role, tier_role = await self.team_manager_cog._roles_for_team(ctx, team_name)
+        trans_channel = await self._trans_channel(ctx)
+        gm_name = self._get_gm_name(ctx, franchise_role)
+        message = "{0} was re-signed by the {1} ({2} - {3})".format(
+            user.mention, team_name, gm_name, tier_role.name)
+
+        if franchise_role in user.roles and tier_role in user.roles:
+            trans_channel.send(message)
```

Comment:
This would cause errors because the `.send()` operation requires an `await` preceding keyword. But as per the comment above, the block in its entirety appears unnecessary.

Reply:
Fixed.